SafeguardSessions/Src/SafeguardSessions.pq

```diff
@@ -89,12 +93,12 @@ shared GetData = (
     in
         result;
 
-ValidateSPSVersion = (
-    spsUrl as text, sessionId as text, skipVersionCheck as logical, optional mocks as nullable record
+shared ValidateSPSVersion = (
+    spsUrl as text, sessionId as text, skipVersionCheck as nullable text, optional mocks as nullable record
 ) =>
     let
         sessionIdOrError =
-            if skipVersionCheck then
+            if skipVersionCheck = Yes then
```

Comment:
Right now if the user does not use the Power BI desktop's input parameter window to select an allowed value for `skipVersionCheck` but rather uses Transform data, any text value can be provided for `skipVersionCheck` and since there is no check that the user provides an allowed value, if the value is other than Yes, version check will be performed instead of raising an error that indicates the possible values for `skipVersionCheck`.

We have agreed that we won't fix it in this PR, but fix it a bug ticket later, can you link here the azure item when you have created it successfully?

Reply:
The related bug is this: https://dev.azure.com/1id/SPS/_workitems/edit/438593